addons/units/functions/fnc_altInfo.sqf

```diff
@@ -0,0 +1,7 @@
+#include "script_component.hpp"
+
+params ["_unit"];
+
+if (_unit isEqualTo ALT_F) exitWith {["f", 0.3047999902464003]};
```

Comment:
Should probably be a macro with descriptive name. (It is the ratio of meters per feet).